pom.xml

```diff
@@ -109,6 +111,52 @@
                     <scmCommentPrefix xml:space="preserve">MOJITO RELEASE: </scmCommentPrefix>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.24.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>apply</goal>
+                        </goals>
+                        <phase>compile</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <pom>
+                        <sortPom>
+                            <nrOfIndentSpace>4</nrOfIndentSpace>
+                        </sortPom>
+                    </pom>
+                    <java>
+                        <googleJavaFormat>
+                            <version>${plugin.version.google-java-format}</version>
+                            <style>GOOGLE</style>
+                        </googleJavaFormat>
+                        <importOrder>
+                            <wildcardsLast>false</wildcardsLast>
+                            <order>com.box,com,org,javax,java,,\#,\\#</order>
```

Comment:
isn't the order automatic?

Reply:
The default one is quite different than what we use today - if you try it locally you'll see what I mean.

If we're happy with the default style the plugin offers - I'm Ok to rely on that.

Reply:
the auto-formatter might be better than what we have today.  Actually I'm a bit surprise with our current order. In my mind, I have this reference

```
4.2 The import statements must follow the package statement. import statements should be sorted with the most fundamental packages first, and grouped with associated packages together and one blank line between groups.
```

this doesn't seem to be followed today, and maybe the formatter fixed it to follow those rules?

Reply:
I agree with the intent - however what the plugin does without that - is that it separates static imports - and everything else seems to be sorted alphabetically into one block without any spaces in between, e.g. for BranchDeleteCommand this is the result:

`package com.box.l10n.mojito.cli.command;

import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_CREATED_BEFORE_LAST_WEEK_DESCRIPTION;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_CREATED_BEFORE_LAST_WEEK_LONG;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_CREATED_BEFORE_LAST_WEEK_SHORT;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_NULL_BRANCH_DESCRIPTION;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_NULL_BRANCH_LONG;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_NULL_BRANCH_SHORT;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_TRANSLATED_DESCRIPTION;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_TRANSLATED_LONG;
import static com.box.l10n.mojito.cli.command.param.Param.BRANCH_TRANSLATED_SHORT;

import com.beust.jcommander.Parameter;
import com.beust.jcommander.Parameters;
import com.box.l10n.mojito.cli.command.param.Param;
import com.box.l10n.mojito.cli.console.ConsoleWriter;
import com.box.l10n.mojito.rest.client.AssetClient;
import com.box.l10n.mojito.rest.client.RepositoryClient;
import com.box.l10n.mojito.rest.entity.Branch;
import com.box.l10n.mojito.rest.entity.PollableTask;
import com.box.l10n.mojito.rest.entity.Repository;
import java.util.List;
import org.fusesource.jansi.Ansi;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;
`

Reply:
Actually, looking at the style guide that seems to match the intent for the most part:

> 3.3.3 Ordering and spacing
> Imports are ordered as follows:
> 
> All static imports in a single block.
> All non-static imports in a single block.
> If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.
> 
> Within each block the imported names appear in ASCII sort order. (Note: this is not the same as the import statements being in ASCII sort order, since '.' sorts before ';'.)